### PR TITLE
Add EmailSignup component and add to Footer

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Adds EmailSignup component
- Adds EmailSignup to Footer
- Also fixes some Safari height issues I noticed in #47

### Type(s) of changes

- [x] New feature
- [x] Update to an existing feature

### Motivation for PR

#20 
### Applicable screenshots:

image
image

### CODE REVIEW

**Comment by chrismekim** [comment]
@mokaymokay @limsohee1002 I'll ping you when the dependency PR is merged :)

**Comment by limsohee1002** [comment]
@chrismekim need RB :)

File: src/components/EmailSignup.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      return errors;
+    },
+    onSubmit: (values: FormikValues) => {},
+  });
+
+  return (
+    <div className="EmailSignup col-12 flex relative">
+      <form
+        noValidate
+        action="https://computer.us11.list-manage.com/subscribe/post?u=6e62b74d002f42a0e5350892e&amp;id=4f35a74dc5"
```
Comment (limsohee1002) [inline src/components/EmailSignup.tsx:36]: Might be better to put this in env file.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -35,66 +34,66 @@ const Footer: React.FC<Props> = ({ globalSettings }) => {
...
+                <div className="Footer__sublinks flex flex-col">
+                  {footerMenu.aboutLinks.map((link: IButton) => (
+                    <Button
+                      className="color-charcoal transition-shorter hover-lighten-charcoal secondary-bold-sm items-center bg-color-transparent text-decoration-none"
+                      ariaLabel={Language.t('Global.generalButtonAriaLabel', {
+                        link: link.label,
+                      })}
+                      to={link.link}
+                      label={link.label}
+                    />
```
Comment (limsohee1002) [inline src/components/Footer.tsx:66]: Need to add key here.

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -35,66 +34,66 @@ const Footer: React.FC<Props> = ({ globalSettings }) => {
...
+                <div className="Footer__sublinks flex flex-col">
+                  {footerMenu.whyWeInvestLinks.map((link: IButton) => (
+                    <Button
+                      className="color-charcoal transition-shorter hover-lighten-charcoal  secondary-bold-sm items-center bg-color-transparent text-decoration-none inline"
+                      ariaLabel={Language.t('Global.generalButtonAriaLabel', {
+                        link: link.label,
+                      })}
+                      to={link.link}
+                      label={link.label}
+                    />
```
Comment (limsohee1002) [inline src/components/Footer.tsx:87]: Add key :)

**Review by limsohee1002 — APPROVED** [review]

File: src/components/Footer.tsx (hunk 2 above)
Comment (chrismekim) [reply src/components/Footer.tsx:66]: I can't believe I keep forgetting the keys!! 🙈  Thank yoU!

**Review by mokaymokay — APPROVED** [review]

**Pull request merged**